schema/core/primitive/acyclic_graph/named_node_in_group.json

```diff
@@ -0,0 +1,28 @@
+{
+    "schemaId": "core-primitive-graph-nodes-named-group",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Named node in group schema",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "named_node.json"
+        },
+        {
+            "$ref": "#/definitions/group-info"
+        }
+    ],
+    "definitions": {
```

Comment:
I think this definition deserves to be in its own file at the top level of the `primitive` folder, no?

Reply:
Since we have plans to put it to use in other places, yes. Although generally, I think it is perhaps better to wait to extract definitions like this to separate files until they can be reused in at least one other schema.